recipe/conda_build_config.yaml

```diff
@@ -4,6 +4,49 @@ c_stdlib_version:  # [osx and x86_64]
 MACOSX_SDK_VERSION:         # [osx]
   - 12.3                    # [osx]
 
+# Needs higher stdlib for CUDNN9
+# CUDA 12 on aarch needs newer glibc, which requires us to override
+# the whole monster zip just to bump c_stdlib_version / docker_image
+c_stdlib_version:                                   # [linux and aarch64]
+  - 2.17                                            # [linux and aarch64]
+  - 2.17                                            # [linux and aarch64]
+  - 2.28                                            # [linux and aarch64]
+docker_image:                                       # [linux and aarch64]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64      # [linux and aarch64]
+  - quay.io/condaforge/linux-anvil-cuda:11.8        # [linux and aarch64]
+  - quay.io/condaforge/linux-anvil-alma-x86_64:8    # [linux and aarch64]
+# rest just to make smithy happy
+c_compiler_version:                                 # [linux and aarch64]
+  - 13                                              # [linux and aarch64]
+  - 11                                              # [linux and aarch64]
+  - 12                                              # [linux and aarch64]
+cxx_compiler_version:                               # [linux and aarch64]
+  - 13                                              # [linux and aarch64]
+  - 11                                              # [linux and aarch64]
+  - 12                                              # [linux and aarch64]
+fortran_compiler_version:                           # [linux and aarch64]
+  - 13                                              # [linux and aarch64]
+  - 11                                              # [linux and aarch64]
+  - 12                                              # [linux and aarch64]
+cdt_name:                                           # [linux and aarch64]
+  - cos7                                            # [linux and aarch64]
+  - cos7                                            # [linux and aarch64]
+  - cos7                                            # [linux and aarch64]
+cuda_compiler:                                      # [linux and aarch64]
+  - None                                            # [linux and aarch64]
+  - nvcc                                            # [linux and aarch64]
+  - cuda-nvcc                                       # [linux and aarch64]
+cuda_compiler_version:                              # [linux and aarch64]
+  - None                                            # [linux and aarch64]
+  - 11.8                                            # [linux and aarch64]
+  - 12.0                                            # [linux and aarch64]
+# end override of monster zip
+
+cuda_compiler_version:                              # [linux64]
+  - None                                            # [linux64]
+  - 11.8                                            # [linux64]
+  - 12.0                                            # [linux64]
```

Comment:
```suggestion
  - 12.2                                            # [linux64]
```

Reply:
you might need to replicate the rest of everything too.

Reply:
Rerendering with 12.2 seems to fail per:

> If there is a recipe/conda-build-config.yaml file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).